src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -2285,11 +2288,15 @@ public function validateString($attribute, $value)
      *
      * @param  string  $attribute
      * @param  mixed  $value
+     * @param  array<string, null|string> $parameters
      * @return bool
      */
-    public function validateTimezone($attribute, $value)
+    public function validateTimezone($attribute, $value, $parameters)
```

Comment:
I think this needs a default value of null.

`$parameters = null`

Reply:
Probably better to use `= []`

Reply:
I am not sure about this. I've checked other validation rules and the only one has a default parameter and it is not null but an empty array. So if I should add a default value I think it should be an empty array not null.

```php
$parameters = []
```

But I think it is not required

Reply:
Done! Thanks.